ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -239,10 +239,8 @@ instance ShelleyCompatible proto era => QueryLedger (ShelleyBlock proto era) whe
           getProposedPPUpdates st
         GetStakeDistribution ->
           SL.poolsByTotalStakeFraction globals st
-        GetUTxOByAddress addrs ->
-          SL.getFilteredUTxO st addrs
-        GetUTxOWhole -> -- FIXME:  if we never match this case then we should probaly clarify this to avoid confusions when reading this part of the code.
-          SL.getUTxO st
+        GetUTxOByAddress{} -> error "Unreachable"
```

Comment:
This can be reached if we define some instances. Maybe we should explain why we shouldn't reach this case.

Reply:
Rephrased the `error` message